src/selectors/index.js

```diff
@@ -0,0 +1 @@
+export { default as exists } from './exists'
```

Comment:
I'm not sure if there was a discussion to follow this pattern of having a map file for selectors. 
@raineorshine, what's your opinion?

Reply:
I'm agnostic. An index creates cleaner bulk imports, but involves manually maintaining a separate file and it makes refactoring individual imports less automatable. 

@umair-khanzada originally suggested the index file, so perhaps he has more reasons he can offer if he has a strong opinion.

Reply:
Yes, I had suggested this, there is a couple of benefits of adopting this technic
1. React and other most popular lib using the same structure ie:(`{Fragment, Component, PureComponent, etc}`) 
2. Create a public interface for accessing a file
3. Import more than one selector in a single statement ie:(`{S1, S2} from 'selectors'`)
4. We are doing the same thing for util and sometimes it is confusing because both have the same name